fe/fe-core/src/main/java/org/apache/doris/journal/bdbje/BDBEnvironment.java

```diff
@@ -207,7 +207,8 @@ public void setup(File envHome, String selfNodeName, String selfNodeHostPort,
                         LOG.warn("", e1);
                     }
                 } else {
-                    LOG.error("error to open replicated environment. will exit.", e);
+                    LOG.error("error to open replicated environment,"
+                            + "Maybe it's due to insufficient disk. will exit.", e);
```

Comment:
The image exception indicate the bdbje state is unknown and disallow to write,  adding a `insufficient disk` log infomation here may be confused in other exception.

Reply:
> The image exception indicate the bdbje state is unknown and disallow to write, adding a `insufficient disk` log infomation here may be confused when met other exception.

This is just to provide users with an idea to solve the problem. In fact, I couldn't get the status due to insufficient disk.